lcm-python/setup.py

```diff
@@ -23,8 +23,19 @@
     ]
 
 
+lcm_version_info = {}
+with open(os.path.join("..", "lcm", "lcm_version.h"), 'r') as lcm_version_file:
+    for line in lcm_version_file:
+        if line.startswith('#define LCM_VERSION'):
+            parts = line.strip().split()
+            lcm_version_info[parts[1]] = parts[2]
+
+lcm_version = \
+    '%(LCM_VERSION_MAJOR)s.%(LCM_VERSION_MINOR)s.%(LCM_VERSION_PATCH)s' \
+    % lcm_version_info
```

Comment:
I think I might be able to emulate something like this for the `rockspec`. (Probably overdue.)